hathor/indexes/manager.py

```diff
@@ -186,16 +292,19 @@ def __init__(self, db: 'rocksdb.DB') -> None:
         self.block_tips = TipsIndex()
         self.tx_tips = TipsIndex()
 
-        self.sorted_all = RocksDBTimestampIndex(self._db, cf_name=b'timestamp-sorted-all')
-        self.sorted_blocks = RocksDBTimestampIndex(self._db, cf_name=b'timestamp-sorted-blocks')
-        self.sorted_txs = RocksDBTimestampIndex(self._db, cf_name=b'timestamp-sorted-txs')
+        self.sorted_all = RocksDBTimestampIndex(self._db, 'all')
+        self.sorted_blocks = RocksDBTimestampIndex(self._db, 'blocks')
+        self.sorted_txs = RocksDBTimestampIndex(self._db, 'txs')
 
         self.addresses = None
         self.tokens = None
         self.height = RocksDBHeightIndex(self._db)
-        self.mempool_tips = RocksDBMempoolTipsIndex(self._db)
+        self.mempool_tips = MemoryMempoolTipsIndex()  # use of RocksDBMempoolTipsIndex is very slow and was suspended
```

Comment:
How much slow?

Reply:
I'll have to run again to confirm, from what I remember, on my machine, using the rocksdb indexes on everything _except_ the mempool (including using the rocksdb-deps index, which are not used by default) it processed at an average rate of ~2000 tx/s, by only adding the rocksdb-mempool-tips-index (instead of the memory implementation), this rate fell to around ~150 tx/s (more than a 90% slowdown), which makes it take 10x longer to load.

Because of how the mempool-tips-index works, I think it should be able to have a specialized implementation that does not work by having a method called on "every" transaction, but instead will either make a walk or use other indexes _after_ they have been initialized.

Reply:
Got it. I feel we should create an issue to investigate this slowness. Please add these numbers to the issue as a starting point for the investigation.

Reply:
OK.

Reply:
Done: #419

Reply:
Thanks!